docs-user/js/init.js

```diff
@@ -1,5 +1,5 @@
 window.$docsify = {
-  name: 'perf.html docs',
+  name: 'Firefox Profiler docs',
```

Comment:
![image](https://user-images.githubusercontent.com/454175/52351758-aa432c00-2a2b-11e9-9882-38256623496c.png)

This is a bit long here :(